proto/prehog/v1alpha/teleport.proto

```diff
@@ -657,6 +685,9 @@ message KubeRequestEvent {
   //
   // PostHog property: tp.user_name
   string user_name = 1;
+
+  // Indicates the kind of user (e.g. bot or human), if known.
+  UserKind user_kind = 2;
```

Comment:
Are we sure we don't want a string here? We seem to be using strings (for the most part) for enum-y things here in the streamed events.